docs/UserGuide.md

```diff
@@ -524,6 +531,7 @@ database.
 <div style="border: 0px solid #ccc; background-color: #d9edff; color: darkblue; padding: 10px; margin-bottom: 10px;">
 Note: The prefixes vary across different modes. Refer to the examples below for more clarity.
 </div>
+Here are some examples of how the command works in each mode:
```

Comment:
This line can be removed since its similar to the one in the callout just above it